tests/fixtures/geojson_to_json/basic_1.expected.json

```diff
@@ -0,0 +1,100 @@
+                    "location": {
+                        "type": "Point",
+                        "coordinates": [
+                            -0.174,
+                            5.625
+                        ]
+                    }
+                },
+                {
+                    "id": "2",
+                    "name": "Kumasi",
+                    "status": "operational",
+                    "location": {
+                        "type": "Point",
+                        "coordinates": [
+                            -1.628,
+                            6.711
+                        ]
+                    }
+                }
+            ],
+            "spans": [
+                {
+                    "id": "1",
+                    "name": "Accra to Kumasi",
+                    "start": "1",
+                    "end": "2",
+                    "route": {
+                        "type": "LineString",
+                        "coordinates": [
+                            [
+                                -0.173,
+                                5.626
+                            ],
+                            [
+                                -0.178,
+                                5.807
+                            ],
+                            [
+                                -0.112,
+                                5.971
+                            ],
+                            [
+                                -0.211,
+                                5.963
+                            ],
+                            [
+                                -0.321,
+                                6.17
+                            ],
+                            [
+                                -0.488,
+                                6.29
+                            ],
+                            [
+                                -0.56,
+                                6.421
+                            ],
+                            [
+                                -0.752,
+                                6.533
+                            ],
+                            [
+                                -0.867,
+                                6.607
+                            ],
+                            [
+                                -1.101,
+                                6.585
+                            ],
+                            [
+                                -1.304,
+                                6.623
+                            ],
+                            [
+                                -1.461,
+                                6.727
+                            ],
+                            [
+                                -1.628,
+                                6.713
+                            ]
+                        ]
+                    }
+                }
+            ],
+            "phases": [],
+            "organisations": []
```

Comment:
This will fail validation because the schema sets `minItems` on these arrays. Empty arrays (and any field with no value) should be omitted in their entirety.

Reply:
Thanks, fixed!